frontend/pages/hosts/details/HostDetailsPage/modals/RunScriptModal/_styles.scss

```diff
@@ -28,6 +28,7 @@
   .Select-option {
     .dropdown__option {
       [data-id="tooltip"] {
```

Comment:
Would `@include tooltip-text` also ensure all other properties are consistent with other tooltips? Or maybe this an edge situation of some sort.

Reply:
Good idea. The mix-in doesn't set the `font-style` though maybe it should? In this case, the select dropdown disabled styles end up making it italicized.

Reply:
We actually have the `DropdownOptionTooltipWrapper` for this functionality, will that work here?

Looks like we're including a `font-style` in some places and not others (e.g. in `TooltipWrapper`), but if we want that to be the same for all of them then I agree it makes sense to include it in the `tooltip-text` mixin, yea.

Reply:
@jacobshandling, it doesn't look like the `DropdownOptionTooltipWrapper` is actually being used anywhere in the frontend or am I missing something?

Reply:
@gillespi314 it's used in `Dropdown`...yet another reason to convert it to `.tsx` 😅